File: Wei.Web/app/model/sys/PropertyValue.js
Ext.define('Wei.model.sys.PropertyValue', {
    extend: 'Wei.model.Base',

    fields: [
        { type: 'int', name: 'id', persist: true },
        { type: 'int', name: 'propertyid', persist: true },
        { type: 'string', name: 'text', persist: true },
        { type: 'int', name: 'isdef', persist: true },
        { type: 'int', name: 'sort', persist: true }

    ]
});

File: Wei.Core/Domain/Sys/PropertyValue.cs
namespace Wei.Core.Domain.Sys
{
    public class PropertyValue: BaseEntity
    {
        public int PropertyId { get; set; }
        public virtual Property Property { get; set; }
        public string Text { get; set; }
        public int IsDef { get; set; }
        public int Sort { get; set; }
    }
}
